services/QuillLMS/app/controllers/teachers/progress_reports/activity_sessions_controller.rb

```diff
@@ -134,7 +140,7 @@ def index
       csv_string = CSV.generate do |csv|
         csv << ['Student', 'Date', 'Activity', 'Score', 'Standard', 'Tool']
         activity_sessions.map do |session|
-          score = (session['percentage'].to_f == -1.0 ? 'Completed' : "#{session['percentage'].to_f*100}%")
+          score = (session['percentage'] == -1.0 ? 'Completed' : "#{session['percentage'] * 100}%")
```

Comment:
Looking at this query, `(CASE WHEN activity_classifications.scored THEN activity_sessions.percentage ELSE -1 END) AS percentage`, bad data could leave `percentage` as `nil`, so that `to_f` might also be a `nil` cover. Might be worth leaving here or confirming that doesn't happen in the data.

Reply:
Good point.  I'm working on a fixing a potentially similar thing with FRL here:
![Screen Shot 2021-05-30 at 7 05 33 AM](https://user-images.githubusercontent.com/2057805/120101739-8680e400-c115-11eb-97e5-7a179d430910.png)

Reply:
I see the issue.  We're basically coercing nils to 0%.  Wouldn't it be more appropriate to leave blank?

Reply:
It might be more appropriate, depends on the report usage and tool (that's more of a product decision). To keep this PR scoped, it might be better to keep the functionality as it is, and to make a new task card to determine if that product change (0% -> blank) should be made (You can add it to 'Tasks').

Reply:
I'll leave the existing functionality then.